src/main/java/com/apaliakou/Consequent.java

```diff
@@ -0,0 +1,176 @@
+package main.java.com.apaliakou;
+
+import java.util.SortedMap;
+
+class Data { // создал класс для исходных данных
+    void showmenu () { // создал метод отображения исходного меню
```

Comment:
Method signuture in java like this [private/no modifier/protected/public/] [static] [abstract] [final] [synchronized] [native] [<T>] ReturnType nameMethod(parameters) [throws exception]

void showMenu() { ....}